### DISCUSSION TITLE
Android Studio Hedgehog의 새로운 패치와 관련해서 의견을 구하고자 합니다.

### DISCUSSION THREAD

**Discussion by koreandroid** [sesac-team-triad/for-pets-android#30] [Tech Consideration]
아까 전에 개발하다가 Android Studio에 뜬 알림을 보고 업데이트를 진행하였습니다.

그런데 업데이트 후, Android Gradle Plugin의 버전이 바뀌었는지 루트 디렉토리의 build.gradle.kts 파일이 변경되었습니다.

보아하니 8.2.1 버전에서 8.2.2 버전으로 바뀐 것 같은데, Android Studio 사용이 처음이라 이와 같은 일이 발생할 것을 예측하지 못 했습니다.

다들 개발하시느라 바쁜 와중에 정말 죄송하지만, 어떻게 하는 것이 좋을지 의논드리고자 글을 작성하게 되었습니다.

가능할지는 잘 모르겠지만, 제가 Android Studio의 버전을 롤백하는 방법과 나머지 두 분께서 Hedgehog 2023.1.1 Patch 2로 업데이트 해주신 후 프로젝트를 이어 나가는 방법들 중에서 저희끼리 결정해야 할 것 같은데, 두 분의 의견이 궁금합니다!

변경된 build.gradle.kts 파일과 새로운 패치에 관한 정보도 함께 공유드리도록 하겠습니다.

@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-&nbsp;id("com.android.application") version "8.2.1" apply false
+&nbsp;id("com.android.application") version "8.2.2" apply false
&nbsp;&nbsp;id("org.jetbrains.kotlin.android") version "1.9.10" apply false
&nbsp;&nbsp;id("com.google.gms.google-services") version "4.4.0" apply false
<br>
https://androidstudio.googleblog.com/2024/01/android-studio-hedgehog-202311-patch-2.html

**Comment by Ju-lotty** [comment]
전 상관 없습니다~~

**Comment by kska01** [comment]
오늘 머지 하기 전에 patch 2로 업데이트 하면 괜찮을 것 같습니다!!
